Sources/kwarchart/charts/PieChart/PieChartDataModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PieChartDataModel.swift
+//  kwarchart
+//
+//  Created by Amiel Jireh Cordova on 5/27/21.
+//
+
+import Foundation
+import SwiftUI
+
+@available(iOS 13.0, *)
+final class PieChartDataModel: ObservableObject {
+    var chartCellModels: [PieSeries] = []
+    var startingAngle = Angle(degrees: 0)
+    private var finalRelativeAngle = Angle(degrees: 0)
+    
+    init(dataModel: [PieSeries]) {
+        chartCellModels = dataModel
+    }
+    
+    var totalValue: CGFloat {
+        chartCellModels.reduce(CGFloat(0)) { (result, data) -> CGFloat in
```

Comment:
```suggestion
        chartCellModels.map { $0.data.data.value }.reduce(0, +)
```
something like that, not sure with the proper fields inside the map func